modules/data-entry/src/main/frontend/src/dataHomepage/ExportButton.jsx

```diff
@@ -0,0 +1,395 @@
+//
+//  Licensed to the Apache Software Foundation (ASF) under one
+//  or more contributor license agreements.  See the NOTICE file
+//  distributed with this work for additional information
+//  regarding copyright ownership.  The ASF licenses this file
+//  to you under the Apache License, Version 2.0 (the
+//  "License"); you may not use this file except in compliance
+//  with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing,
+//  software distributed under the License is distributed on an
+//  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+//  KIND, either express or implied.  See the License for the
+//  specific language governing permissions and limitations
+//  under the License.
+//
+import React, { useState, useEffect } from 'react';
+import PropTypes from "prop-types";
+import { makeStyles } from '@mui/styles';
+import { deepPurple, orange } from '@mui/material/colors';
+import { Avatar, Checkbox, Dialog, DialogTitle, DialogActions, DialogContent, FormControl, Icon, Grid, Radio, RadioGroup,
+  FormControlLabel, Typography, Button, IconButton, Tooltip, InputLabel, Select, ListItemText, MenuItem } from "@mui/material";
+import { FileDownload } from "@mui/icons-material";
+import CloseIcon from '@mui/icons-material/Close';
+
+
+const useStyles = makeStyles(theme => ({
+  container: {
+    "& .MuiGrid-item:not(:first-child)": {
```

Comment:
There should be spacing between containers, and the label of the last one (list of questions/sections) should have some top padding for better alignment.
```suggestion
    paddingTop: theme.spacing(2),
    "&:last-child > .MuiGrid-item:first-child" : {
```